<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AP2DataPlotAxisDialog</class>
 <widget class="QWidget" name="AP2DataPlotAxisDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Graph Scale Control</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>&lt;h1&gt;Axis Range Grouping&lt;/h1&gt;
This Dialog is used for grouping multiple graphs with a single value range (the axis on the left hand side). Select a graph name from the list, then click on the Axis group you wish it to be in. Multiple graphs in the same group will all have the same range, which will be automatically calculated to constrain every graph in the group</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,2,3">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QTableWidget" name="graphTableWidget"/>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Axis Range Control</string>
       </property>
       <widget class="QRadioButton" name="groupDRadioButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>140</y>
          <width>82</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Group D</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="groupARadioButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>82</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Group A</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="autoRadioButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>101</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Automatic Axis</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="groupCRadioButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>110</y>
          <width>82</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Group C</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="groupBRadioButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>82</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Group B</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="manualRadioButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>170</y>
          <width>82</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Manual</string>
        </property>
       </widget>
       <widget class="QWidget" name="horizontalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>200</y>
          <width>161</width>
          <height>22</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Scale Graph:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="graphScaleDoubleSpinBox">
           <property name="decimals">
            <number>2</number>
           </property>
           <property name="maximum">
            <double>100000.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="minMaxGroupBox">
       <property name="title">
        <string>Manual Min/Max Control</string>
       </property>
       <widget class="QDoubleSpinBox" name="maxDoubleSpinBox">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>20</y>
          <width>131</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimum">
         <double>-100000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>100000.000000000000000</double>
        </property>
       </widget>
       <widget class="QPushButton" name="setMinMaxPushButton">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>90</y>
          <width>101</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Set New Min/Max</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>46</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Maximum</string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="minDoubleSpinBox">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>60</y>
          <width>131</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimum">
         <double>-100000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>100000.000000000000000</double>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>46</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Minimum</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="applyPushButton">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelPushButton">
       <property name="text">
        <string>Done</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
